app/controllers/admin/vote_users_controller.rb

```diff
@@ -22,6 +22,8 @@ def update
 
   def show
     @user = User.find(params[:id])
+    @audit_grid = initialize_grid(@user.audits, name: 'g1')
+    @assoc_grid = initialize_grid(VotePost.includes(:audits).where(user_id: @user.id), name: 'g2')
```

Comment:
It does easily get messy with three instance variables from one action.
It could be worth creating some kind of view_object.